gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Wed Jan 22 15:25:48 PST 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1-all.zip
```

Comment:
are there any breaking changes with gradle 6?

Reply:
There are deprecated APIs in Gradle 6, for example in publishing the "classifier" attributed had to be changed.